themes/base/tooltip.css

```diff
@@ -13,8 +13,8 @@
 	position: absolute;
 	z-index: 9999;
 	max-width: 300px;
-	-webkit-box-shadow: 0 0 5px #aaa;
-	box-shadow: 0 0 5px #aaa;
+	-webkit-box-shadow: 0 0 5px #ddd;
+	box-shadow: 0 0 5px #ddd;
```

Comment:
In today's UI meeting, it was decided to "Update tooltip to use ui-widget-shadow class (via classes option, once that lands), move box-shodow definition into theme.css/.ui-widget-shadow, update TR to demo that instead of the weird fake shadow we have currently"